src/platforms/pumpfun/curve_manager.py

```
@@ -0,0 +1,322 @@
+            ]
+            
+            for field in required_fields:
+                if field not in pool_state:
+                    logger.error(f"Missing required field: {field}")
+                    return False
+                
+                if field != "complete" and not isinstance(pool_state[field], int):
+                    logger.error(f"Field {field} is not an integer: {type(pool_state[field])}")
+                    return False
+            
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)